src/main/java/com/softserveinc/dokazovi/controller/EndPoints.java

```diff
@@ -4,5 +4,6 @@
 
 	public static final String WEB_API = "/api";
 	public static final String VERSION = "/version";
+	public static final String IMPORTANT = "/important";
```

Comment:
Please use the hierarchy of api:
public static final String POSTS = "/posts";
public static final String IMPORTANTN_POSTS = POSTS  + "/important";